Switch_Client/source/main.c

```diff
@@ -126,6 +127,15 @@ int main(int argc, char* argv[]) {
 		if (kDown & KEY_DUP && currentIpBlock == 3) ipBlock4++;
 		if (kDown & KEY_DDOWN && currentIpBlock == 3) ipBlock4--;
 
+	    	if (kDown & KEY_ZR && currentIpBlock == 0) ipBlock1+=10;
+		if (kDown & KEY_ZL && currentIpBlock == 0) ipBlock1-=10;
+		if (kDown & KEY_ZR && currentIpBlock == 1) ipBlock2+=10;
+		if (kDown & KEY_ZL && currentIpBlock == 1) ipBlock2-=10;
+		if (kDown & KEY_ZR && currentIpBlock == 2) ipBlock3+=10;
+		if (kDown & KEY_ZL && currentIpBlock == 2) ipBlock3-=10;
+		if (kDown & KEY_ZR && currentIpBlock == 3) ipBlock4+=10;
+		if (kDown & KEY_ZL && currentIpBlock == 3) ipBlock4-=10;
```

Comment:
another sidenote, this monster of an if() chain could definitely be cut down a bit, but i'm kind of lazy... and also it only needs to run until the switch connects to the IP in question, so it should be fine